self_driving/vehicle_state_reader.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, vehicle: Vehicle, beamng: BeamNGpy, additional_sensors: List[
         self.vehicle.attach_sensor('electrics', electrics)
         self.vehicle.attach_sensor('timer', timer)
 
+        #TODO
```

Comment:
Probably we should get rid of this one...

Reply:
probably yes